app/design/frontend/Magento/blank/Magento_Swatches/web/css/source/_module.less

```diff
@@ -1,6 +1,6 @@
 // /**
-//  * Copyright © Magento, Inc. All rights reserved.
-//  * See COPYING.txt for license details.
+//  * Copyright 2024 Adobe
```

Comment:
Year in copyright header should be the year the file was first added in git, so in this case 2017 instead of 2024

Reply:
@Serfe-com: this one as well please, it looks like you reverted it back to 2024 in your last commit, thanks! :)